### ISSUE TITLE
[FEATURE] When using the view, I noticed that the vertical scrolling area is quite large. Is it possible to limit the scrollable range?

### ISSUE THREAD

**Issue by KWYHDHJ** [frangoteam/FUXA#1547]
**Describe the feature**
I often accidentally scroll with the mouse wheel while using it, which results in only being able to see a small portion of the view.

**Describe the solution you'd like**
Is it possible to restrict the range of the negative ruler in the view?

**Additional context**
![PixPin_2024-12-23_10-44-09](https://github.com/user-attachments/assets/7df11e2e-da91-42ca-a2ae-a5ce01e2276d)
![PixPin_2024-12-23_10-36-01](https://github.com/user-attachments/assets/2060237d-da98-4bfe-afab-3cb2d59cba1a)

**Comment by KWYHDHJ** [comment]
Is the view editing a method you encapsulated yourself? I tried modifying the corresponding code in the client folder but couldn't find where the settings are located.

**Comment by unocelli** [comment]
Hi, you can zoom In to reduce it. 
Shift + SCROLLER: zoom by mouse position

**Comment by KWYHDHJ** [comment]
> Hi, you can zoom In to reduce it. Shift + SCROLLER: zoom by mouse position

Yes, I know that the view size can be zoomed by using Shift + SCROLLER: zoom by mouse position, but wouldn't it be better to add some restrictions outside the view editing area, apart from the white content? After all, we don't use the parts outside the view editing area, so adding some restrictions could optimize the user experience.
